NoSQLTask/src/main/java/com/instinctools/egor/mentoring/noSQL/dal/mongo/entity/BookMongoEntity.java

```diff
@@ -0,0 +1,94 @@
+package com.instinctools.egor.mentoring.noSQL.dal.mongo.entity;
+
+import com.instinctools.egor.mentoring.noSQL.core.entity.Book;
+import com.instinctools.egor.mentoring.noSQL.core.entity.User;
+import com.mongodb.BasicDBObject;
+import com.mongodb.DBObject;
+
+import java.util.UUID;
+
+public class BookMongoEntity {
+    private String _id;
+    private String name;
+    private String author;
+    private UserMongoEntity owner;
+
+    public BookMongoEntity() {
+        this._id = UUID.randomUUID().toString();
```

Comment:
seems we don't need this constructor - we create bookentity from book so id is defined in core.book model